### PR TITLE
chore(repo): improve issue acceptance criteria

### PR DESCRIPTION
**Pull request by CyrusVorwald-ICON**
acceptance criteria in issues generally are not well defined

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 name: Bug
 about: Describes a bug
 title: ''
-labels: test
+labels: ''
```
Comment (MuhammedIrfan) [inline .github/ISSUE_TEMPLATE/bug.md:5]:
```suggestion
labels: 'bug'
```

File: .github/ISSUE_TEMPLATE/epic.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 name: Epic
```
Comment (MuhammedIrfan) [inline .github/ISSUE_TEMPLATE/epic.md:2]: We can move this to the planning repo as we plan to keep 1 issue: 1 pull request in this repo

File: .github/ISSUE_TEMPLATE/feature.md [hunk 3]
```diff
@@ -2,7 +2,7 @@
 name: Feature
```
Comment (MuhammedIrfan) [inline .github/ISSUE_TEMPLATE/feature.md:2]: We can move this to the planning repo as we plan to keep 1 issue : 1 pull request in this repo

File: .github/ISSUE_TEMPLATE/test-scenario.md [hunk 4]
```diff
@@ -2,7 +2,7 @@
 name: Test Scenario
```
Comment (MuhammedIrfan) [inline .github/ISSUE_TEMPLATE/test-scenario.md:2]: We can add a placeholder for Bug identified performing this test

File: .github/ISSUE_TEMPLATE/test-scenario.md [hunk 5]
```diff
@@ -2,7 +2,7 @@
...
 about: Describes a test scenario
 title: ''
-labels: test
+labels: ''
```
Comment (MuhammedIrfan) [inline .github/ISSUE_TEMPLATE/test-scenario.md:5]:
```suggestion
labels: 'test'
```

**Review by pragyanshrestha-ibriz — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/user-story.md [hunk 6]
```diff
@@ -17,4 +17,10 @@ As a [persona], I want [goal] so that [reason]
...
 ## Acceptance Criteria
 
-List of requirements to be considered complete.
+- Code is properly documented in accordance with the [documentation guidelines](https://github.com/icon-project/community/blob/main/guidelines/technical/software-development-guidelines.md#documentation)
+- Test coverage is 80% or higher
+- All tests are passed
+- Code is reviewed and approved in accordance with the [review guidelines](https://github.com/icon-project/community/blob/main/guidelines/technical/software-development-guidelines.md#code-reviews)
+- Specify additional functional acceptance criteria to be met here
+- Specify additional non-functional acceptance criteria are met
+- Code is deployed to a specified environment. Can be local, testnet, mainnet, or something else
```
[the remark this answers is not in the document]
Comment (CyrusVorwald-ICON) [reply .github/ISSUE_TEMPLATE/user-story.md:26]: Good point

File: .github/ISSUE_TEMPLATE/feature.md (hunk 3 above)
Comment (CyrusVorwald-ICON) [reply .github/ISSUE_TEMPLATE/feature.md:2]: Agreed

File: .github/ISSUE_TEMPLATE/test-scenario.md (hunk 4 above)
Comment (CyrusVorwald-ICON) [reply .github/ISSUE_TEMPLATE/test-scenario.md:2]: Sorry, I'm not sure what you mean. Can you expand on this?

File: .github/ISSUE_TEMPLATE/user-story.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (CyrusVorwald-ICON) [reply .github/ISSUE_TEMPLATE/user-story.md]: I have added a pull request template to this branch. Please let me know what you think

File: .github/ISSUE_TEMPLATE/test-scenario.md (hunk 4 above)
Comment (MuhammedIrfan) [reply .github/ISSUE_TEMPLATE/test-scenario.md:2]: I have made the changes

**Review by MuhammedIrfan — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/user-story.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (MuhammedIrfan) [reply .github/ISSUE_TEMPLATE/user-story.md]: Looks good

**Pull request merged**